force-app/main/default/triggers/AccountTrigger.trigger

```diff
@@ -0,0 +1,13 @@
+trigger AccountTrigger on Account (after insert) {
```

Comment:
Some of these should be happening in a before context. Think about it this way.. 

- Is the trigger trying to validate or perform additional updates to the record that caused it to fire? Put this in the before context
- Is the trigger trying to create related records, assign tasks, send emails, or some other process that expects the record that fired has successfully saved? Put this in the after context